cmd/kube-proxy/app/server.go

```diff
@@ -822,7 +823,7 @@ func serveHealthz(logger klog.Logger, hz *healthcheck.ProxierHealthServer, errCh
 			logger.Error(nil, "Healthz server returned without error")
 		}
 	}
-	go wait.Until(fn, 5*time.Second, wait.NeverStop)
+	go wait.Until(fn, 5*time.Second, ctx.Done())
```

Comment:
:+1:

Reply:
:+1: in other places also. I think this is particularly useful combined with `signal.NotifyContext`.